src/types/profile.js

```diff
@@ -185,6 +186,11 @@ export type FrameTable = {|
   category: (IndexIntoCategoryList | null)[],
   subcategory: (IndexIntoSubcategoryListForCategory | null)[],
   func: IndexIntoFuncTable[],
+  // Inner window ID of JS frames. JS frames can be correlated to a Page through this value.
+  // It's used to determine which JS frame belongs to which web page so we can display
+  // that information and filter for single tab profiling.
+  // `0` for others and the JS frames that's been failed to get the ID.
```

Comment:
Thanks, this documentation really helps explain what's going on here.

```suggestion
  // `0` for others and the JS frames that failed to get the ID.
```

This last line is a bit hard to interpret. I think it would be worth elaborating here on what is meant by "others".

Reply:
Changed the "others" with "non-JS".